### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by stellakunzang**
README is completed (mostly, but open for comments and suggestions!) 

Deleted some commented out code. 

Fixed something silly going on with the checkout zip code

### CODE REVIEW

File: spec/models/item_spec.rb [hunk 1]
```diff
@@ -40,10 +38,18 @@
 
     it ".top_5" do
       expect(Item.top_5).to eq({ @dog_bone.name => 8, @pull_toy.name => 5, @tire.name => 4, @toy.name => 3, @bone.name => 2})
+
+      @dog_bone.update(active?: false)
+
+      expect(Item.top_5).to eq({ @pull_toy.name => 5, @tire.name => 4, @toy.name => 3, @bone.name => 2})
```
Comment (stellakunzang) [inline spec/models/item_spec.rb:44]: Oops did this a few days ago on this branch and forgot! Made this test better 👍

File: app/models/item.rb [hunk 2]
```diff
@@ -35,6 +35,7 @@ def default_image
 
   def self.top_5
     Item.joins(:item_orders)
+        .where(active?: true)
```
Comment (stellakunzang) [inline app/models/item.rb:38]: Making sure we are only getting top_5 data from active items

**Review by Gallup93 — APPROVED** [review]
This read me kicks major ass

File: app/models/item.rb (hunk 2 above)
Comment (muydanny) [reply app/models/item.rb:38]: Beautiful

File: README.md [hunk 3]
```diff
@@ -1,1104 +1,212 @@
...
+*Monster Shop* is a web application created by [Sage Lee](https://github.com/sagemlee), [Danny Ramos](https://github.com/muydanny), [Michael Gallup](https://github.com/Gallup93), & [Stella Bonnie](https://github.com/stellakunzang), students at Turing School of Software and Design. It was our final project for Mod 2 (of 4), completed 5 weeks after first encountering Rails, MVC, ActiveRecord, and RSpec.
 
-Students will be put into 3 or 4 person groups to complete the project.\n
+The "Monster Shop" itself is a fictitious e-commerce platform, specializing in silly items. It is something akin to a digital flea market, hosting many different merchants who are each responsible for fulfilling their part of a customer's order. Vistors to the site must be logged-in in order to checkout. Once they enter shipping information and checkout, a merchant user is able to then fulfill their items in the order. Once all of the items from the various merchants have been marked as fulfilled, the order status is changed to "packaged." This status is visible to the customer. A third type of user, an admin or "super user", is then able to mark the order as "shipped."
```
[the remark this answers is not in the document]
Comment (muydanny) [reply README.md:7]: upvote

**Review by muydanny — APPROVED** [review]

**Pull request merged**